plugin/kubernetes/object/service.go

```diff
@@ -40,13 +40,19 @@ func ToService(obj meta.Object) (meta.Object, error) {
 		Name:         svc.GetName(),
 		Namespace:    svc.GetNamespace(),
 		Index:        ServiceKey(svc.GetName(), svc.GetNamespace()),
-		ClusterIP:    svc.Spec.ClusterIP,
 		Type:         svc.Spec.Type,
 		ExternalName: svc.Spec.ExternalName,
 
 		ExternalIPs: make([]string, len(svc.Status.LoadBalancer.Ingress)+len(svc.Spec.ExternalIPs)),
 	}
 
+	if len(svc.Spec.ClusterIPs) > 0 {
+		s.ClusterIPs = make([]string, len(svc.Spec.Ports))
```

Comment:
why  `len(svc.Spec.Ports))` instead of ` len(svc.Spec.ClusterIPs))` ??

Reply:
oops. Thats a mistake. Thanks!